src/main/java/Duke.java

```diff
@@ -6,5 +8,34 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you ?");
+        Scanner scanObj = new Scanner(System.in);
+        TaskManager listofItems = new TaskManager();
+        String userCmd = scanObj.nextLine();
+        int taskId = 0;
+
+        while (!userCmd.equals("bye")) {
+            String[] userCmdasWords = userCmd.split(" ");
+            System.out.println(userCmdasWords[0]);
+            if (userCmd.equals("list")) {
```

Comment:
Perhaps you can consider using switch instead of multiple if-else statements to make your code cleaner and more readable